plugins/modules/cloud/alicloud/ali_instance_info.py

```diff
@@ -35,12 +35,14 @@
 options:
     availability_zone:
       description:
-        - (Deprecated) Aliyun availability zone ID in which to launch the instance. Please use filter item 'zone_id' instead.
+        - Aliyun availability zone ID in which to launch the instance.
+        - Deprecated parameter, it will be removed in community.general C(5.0.0). Please use filter item I(zone_id) instead.
```

Comment:
```suggestion
        - Deprecated parameter, it will be removed in community.general 5.0.0. Please use filter item I(zone_id) instead.
```
Why put version numbers in `C(...)`?

Reply:
no particular reason - I copied this from the `linode` module.

Reply:
solved